File: eval/corpus/disambig/tie_alpha.ts
export function normalize(text: string): string {
  // Lowercase the input and collapse runs of whitespace to single spaces.
  return text.toLowerCase().split(/\s+/).join(" ");
}

export function flatten<T>(nested: T[][]): T[] {
  // Concatenate a list of lists into one flat list, left to right.
  return nested.reduce((acc, sub) => acc.concat(sub), []);
}

export function alphaMarker(): string {
  return "alpha";
}

File: eval/corpus/disambig/tie_alpha.py
def normalize(text):
    """Lowercase the input and collapse runs of whitespace to single spaces."""
    return " ".join(text.lower().split())


def flatten(nested):
    """Concatenate a list of lists into one flat list, left to right."""
    return [item for sub in nested for item in sub]


def alpha_marker():
    return "alpha"
